apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -1,14 +1,17 @@
 /** Body of the animation picker dialog */
 import React from 'react';
 import Radium from 'radium';
+import Immutable from 'immutable';
 import color from '../../color';
 import gamelabMsg from '../locale';
-import animationLibrary from '../animationLibrary';
+import animationLibrary from '../animationLibrary.json';
 import ScrollableList from '../AnimationTab/ScrollableList.jsx';
 import styles from './styles';
 import AnimationPickerListItem from './AnimationPickerListItem.jsx';
 import AnimationPickerSearchBar from './AnimationPickerSearchBar.jsx';
 
+const MAX_SEARCH_RESULTS = 18;
```

Comment:
We are limiting the number of results to a query to avoid triggering a ton of asset loading while-you-type.  Long-term I' like to add pagination to search results so you can keep stepping through animations if you'd like.